nfs-ganesha/build/build_deb

```diff
@@ -36,15 +44,24 @@ cd build
 # make source tarball
 cmake -DCMAKE_BUILD_TYPE=Maintainer -DUSE_FSAL_GLUSTER=OFF -DUSE_FSAL_CEPH=ON -DUSE_FSAL_RGW=ON $WORKSPACE/nfs-ganesha/src && make dist || exit 1
```

Comment:
we need the it to be in a directory called "src" because the debian/rules file looks for a directory with that name (https://github.com/nfs-ganesha/nfs-ganesha-debian/blob/xenial-nfs-ganesha-ceph-fsals/debian/rules#L30). 

I could simplify the process a little bit by removing the .git repo, tar'ing just the "src" by hand, and forgoing the cmake command, but I wanted to keep it similar for consistency. All the steps after that command are confusing though.

Would you recommend any changes?